babel.config.js

```diff
@@ -15,11 +15,6 @@ module.exports = function (api) {
     ].filter(Boolean),
     plugins: [
       process.env.WEBPACK_SERVE && 'react-refresh/babel',
-      isProductionEnv && ['babel-plugin-transform-react-remove-prop-types',
-        {
-          removeImport: true
-        }
-      ]
```

Comment:
why is this no longer necessary?

Reply:
@justin808 Was it necessary before? There are no `prop-types` in this repository. 🤔